src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -526,6 +543,48 @@ abstract class DatabaseTest {
     return row.id!!.also { inserted.projectIds.add(it) }
   }
 
+  private var nextDeliverableNumber: Int = 1
+
+  protected fun insertDeliverable(
+      createdBy: UserId = currentUser().userId,
+      createdTime: Instant = Instant.EPOCH,
+      deliverableCategoryId: DeliverableCategory = DeliverableCategory.FinancialViability,
+      deliverableTypeId: DeliverableType = DeliverableType.Document,
+      descriptionHtml: String? = "",
+      id: DeliverableId = nextDeliverableNumber.toIdWrapper { DeliverableId(it) },
+      isSensitive: Boolean = false,
+      isRequired: Boolean = false,
+      moduleId: ModuleId = inserted.moduleId,
+      name: String = "Deliverable $nextDeliverableNumber",
+      position: Int = 0,
```

Comment:
Positions should start with 1 rather than 0 since `DeliverablesImporter` flips the signs on the positions to detect when there are existing deliverables missing from the imported file.

Using `nextDeliverableNumber` as the default would probably make sense here to avoid position collisions.